cmd/terramate/e2etests/main_test.go

```diff
@@ -146,12 +143,33 @@ func buildTerramate(goBin, projectRoot, binDir string) (string, error) {
 	return outBinPath, nil
 }
 
-func installTerraform() (string, *install.Installer, error) {
+func installTerraform() (string, func(), error) {
+	requireVersion := os.Getenv("TM_TEST_TERRAFORM_REQUIRED_VERSION")
+	tfExecPath, err := exec.LookPath("terraform")
+	if err == nil {
+		cmd := exec.Command("terraform", "version")
```

Comment:
Could use `terraform version -json` here, since the human readable output might change at some point.